spec/jobs/education_form/create_daily_fiscal_year_to_date_report_spec.rb

```diff
@@ -72,9 +72,16 @@ def get_education_form_fixture(filename)
       create(:education_benefits_submission, created_at: date - 26.hours, status: 'processed')
 
       create(:education_benefits_submission, created_at: date, status: 'submitted')
-      %w[1995 1990e 5490 1990n 5495 1995s].each do |form_type|
-        create(:education_benefits_submission, form_type: form_type, created_at: date)
+      if !Flipper.enabled?(:edu_benefits_stem_scholarship, @current_user)
```

Comment:
We shouldn't be checking if `Flipper.enabled?` in the specs.
At this point when the specs run is `Flipper.enabled?(:edu_benefits_stem_scholarship, @current_user)` true or false?
Regardless, there is one path that does not get tested here.

Instead you should set the toggle value to what you want it to be and test as appropriate.
You can search the `spec` directory for `Flipper.enable` to find examples of other devs doing this.  [Here's one](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/lib/facilities/ppms/client_spec.rb).

Reply:
Thank you for sending over the example of that. But yes that is clear I will add in specs for the other case here.